ci-operator/config/jboss-fuse/fuse-xpaas-qe-container/OWNERS

```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- jboss-fuse-approvers
```

Comment:
Please add `cspi-qe-ocp-lp` as the approvers also for all the OWNERS file. Our team is also responsible for maintaining these scenario and should have the rights to approve the changes.